package.json

```diff
@@ -29,16 +29,18 @@
     "moment": "~2.24.0",
     "mout": "~1.1.0",
     "needle": "~2.1.2",
+    "nock": "~12.0.3",
```

Comment:
I think this is meant to be in devDependencies

Reply:
Good catch